python/graphstorm/model/rgcn_encoder.py

```diff
@@ -469,23 +544,33 @@ def forward(self, g, inputs, mod_args=None, mod_kwargs=None):
         outputs = {nty: [] for nty in g.dsttypes}
         if isinstance(inputs, tuple) or g.is_block:
             if isinstance(inputs, tuple):
-                src_inputs, dst_inputs = inputs
+                src_inputs, dst_inputs, edge_inputs = inputs
```

Comment:
This will break the backward compatibility.
We should also handle the case when inputs is (src_inputs, dst_inputs).

Reply:
yes, added an additional check for the backward compatibility.